### PR TITLE
Add ui

### PR DESCRIPTION
**Pull request by MingshiPeng**
How to run:
1. `vagrant up && vagant ssh`
1. `cd /vagrant`
1. `python run.py`
1.  Open a browser and access `localhost:5000`

UI Screenshot:
screen shot 2018-04-24 at 9 51 49 pm

### CODE REVIEW

File: app/server.py [hunk 1]
```diff
@@ -72,22 +72,7 @@ def not_found(error):
...
-            'list all': 'GET /inventory',
-            'read': 'GET /inventory/{prod_id}',
-            'create': 'POST /inventory',
-            'update': 'PUT /inventory/{prod_id}',
-            'delete': 'DELETE /inventory/{prod_id}',
-            'query': 'GET /inventory?{prod_name|quantity|condition=val}',
-            'restock': 'PUT /inventory/{prod_id}/restock',
-        },
-    ), status.HTTP_200_OK
+    return app.send_static_file('index.html')
```
Comment (jockyxu1122) [inline app/server.py]: Does this file contain the link to Swagger docs?

File: app/static/js/rest_api.js [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+
+        var ajax = $.ajax({
+            type: "GET",
+            url: "/inventory/" + prod_id,
+            contentType:"application/json",
+            data: ''
+        })
+
+        ajax.done(function(res){
+            //alert(res.toSource())
```
Comment (jockyxu1122) [inline app/static/js/rest_api.js]: Remove if don't need it

File: app/static/js/rest_api.js [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+
+        var ajax = $.ajax({
+            type: "GET",
+            url: "/inventory?" + queryString,
+            contentType:"application/json",
+            data: ''
+        })
+
+        ajax.done(function(res){
+            //alert(res.toSource())
```
Comment (jockyxu1122) [inline app/static/js/rest_api.js]: remove

**Review by nideyidayida — CHANGES_REQUESTED** [review]

File: app/static/index.html [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+        </div>
+
+        <footer>
+          <br><br>
+          <p>&copy; NYU DevOps Inventory Company 2017</p>
+        </footer>
+      </div><!--/span-->
+    </div><!-- container -->
+  </body>
+</html>
```
Comment (majia67) [inline app/static/index.html]: new line

**Review by majia67 — APPROVED** [review]
LGTM. Please see the comments for nit problems.

**Review by nideyidayida — APPROVED** [review]
good

**Comment by majia67** [comment]
IBM Cloud [toolchain](https://console.ng.bluemix.net/devops/toolchains/d4dd8efb-da17-4b0c-b4e6-5599ec73dd60?env_id=ibm%3Ayp%3Aus-south "nyu-inventory-service-s18"): [Delivery Pipeline](https://console.ng.bluemix.net/devops/pipelines/c71be43a-8963-4052-b97c-5da9b99de3f9?env_id=ibm%3Ayp%3Aus-south "nyu-inventory-service-s18") deployed [inventory](https://github.com/NYUDevOpsInventory/inventory/releases/tag/deploy-ibm-yp-us-south-NYU_Devops_Inventory_Org-dev-20180425-172929 "deploy-ibm-yp-us-south-NYU_Devops_Inventory_Org-dev-20180425-172929") to [dev](https://console.bluemix.net/apps/1db994d8-c4f1-4076-85c0-7ff2674244e5?env_id=ibm:yp:us-south "ibm:yp:us-south:NYU_Devops_Inventory_Org:dev"), including commit 88a06923f5bca90f104e0b390f0e8685514d24b4

**Comment by majia67** [comment]
IBM Cloud [toolchain](https://console.ng.bluemix.net/devops/toolchains/d4dd8efb-da17-4b0c-b4e6-5599ec73dd60?env_id=ibm%3Ayp%3Aus-south "nyu-inventory-service-s18"): [Delivery Pipeline](https://console.ng.bluemix.net/devops/pipelines/c71be43a-8963-4052-b97c-5da9b99de3f9?env_id=ibm%3Ayp%3Aus-south "nyu-inventory-service-s18") deployed [inventory](https://github.com/NYUDevOpsInventory/inventory/releases/tag/deploy-ibm-yp-us-south-NYU_Devops_Inventory_Org-prod-20180501-183630 "deploy-ibm-yp-us-south-NYU_Devops_Inventory_Org-prod-20180501-183630") to [prod](https://console.bluemix.net/apps/e25e653b-a50f-4831-b93e-d8f9f484a484?env_id=ibm:yp:us-south "ibm:yp:us-south:NYU_Devops_Inventory_Org:prod"), including commit 88a06923f5bca90f104e0b390f0e8685514d24b4

**Pull request merged**